tests/e2e/cluster_downgrade_test.go

```diff
@@ -243,3 +253,34 @@ func getMemberVersionByCurl(cfg *e2e.EtcdProcessClusterConfig, member e2e.EtcdPr
 	}
 	return result, nil
 }
+
+func generateSnapshot(t *testing.T, snapshotCount uint64, epc *e2e.EtcdProcessCluster) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	t.Log("Adding and removing keys")
```

Comment:
nit: I only see adding keys, and do not se removing keys:)

Reply:
Thanks, I will update the comment.